layers/erspan2.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"encoding/binary"
+	"net"
+	"time"
+
+	"github.com/google/gopacket"
+)
+
+// Header contains all of the fields found in an ERSPAN2 header
+// https://tools.ietf.org/html/draft-foschiano-erspan-03
+type Header struct {
```

Comment:
This should be called ERPSANIIHeader

Reply:
Ah yes, good point.  I had the different layers separated into their own packages so callers would have `erspan2.Header` or `erspan2.NewHeader()`.  There are a few things, as you've pointed out, than need to be cleaned up a bit to fit into the `layers` package.